ghcup-prereleases-0.0.8.yaml

```diff
@@ -2235,7 +2235,7 @@ ghcupDownloads:
               dlUri: https://downloads.haskell.org/~ghc/9.12.0.20241031/ghc-9.12.0.20241031-x86_64-ubuntu20_04-linux.tar.xz
             unknown_versioning: *ghc9121alpha2-x86_64-ubuntu22_04
           Linux_RedHat:
-            unknown_versioning: *ghc9121alpha2-x86_64-centos7
+            unknown_versioning: *ghc9121alpha2-x86_64-fedora33
```

Comment:
What I meant is:

* centos7 for RedHat 8
* fedora33 for RedHat 9 (and unknown/later)

Reply:
done.